openedx_caliper_tracking/tests/test_caliper_kafka.py

```diff
@@ -248,7 +248,7 @@ def test_sent_kafka_failure_email_with_success(self, send_notification_mock, log
         sent_kafka_failure_email('Dummy Error')
         self.assertTrue(send_notification_mock.called)
         logger_mock.info.assert_called_with('Email Sent Successfully: Events delivery failure report'
-                                            ' sent to dummy@example.com.')
+                                            ' sent to [\'dummy@example.com\'].')
```

Comment:
Can we use a constant for this?

Reply:
We can, but from unit tests' perspective, I think that keeping things simple like this seems more appropriate.